src/activities/content/ContentWebLinkEntity.js

```diff
@@ -68,7 +68,7 @@ export class ContentWebLinkEntity extends Entity {
 		}
 
 		const fields = [{ name: 'description', value: richText }];
-		await performSirenAction(this._token, action, fields);
+		return await performSirenAction(this._token, action, fields);
```

Comment:
good change.. async functions are required to return a Promise iirc. i dont think these functions were really doing the async behavior that it _appeared_ like they were doing.